gramps/plugins/db/bsddb/bsddb.gpr.py

```diff
@@ -17,6 +17,9 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
+from gramps.gen.const import GRAMPS_LOCALE as glocale
+_ = glocale.translation.gettext
```

Comment:
Oh, yes. 

I am also trying to limit [coverage diff](https://github.com/codecov/support/wiki/Coverage-Diff), but sometimes with [unexpected returns](https://github.com/codecov/support/wiki/Unexpected-Coverage-Changes).